tests/test.rs

```diff
@@ -454,3 +454,18 @@ fn triple_nested_structure() {
 
     assert_eq!(expected_values, nested_objects);
 }
+
+#[test]
+fn uncopyable_iterator() {
```

Comment:
Please add a blank line between each one of the items here
```rust
fn uncopyable_iterator() {
    struct Foo {};

    impl Iterator for Foo {
        type Item = ();

        fn next(&mut self) -> Option<()> {
            None
        }
    }

    let _ = comp! {
        x;
        for x in Foo {};
    };
}
```

Reply:
rustfmt wasn't putting them in there :man_shrugging:

Reply:
rustfmt doesn't dictate blank lines inside functions (except that it does not allow more than once at a time) because it doesn't have an opinion about how you logically break up the parts of a function.

It's irrelevant now since you reduced the function to one line anyway 👍